### PR TITLE
feat: [FFM-4072]: New FF Onboarding Landing Page 

### PR DESCRIPTION
**Pull request by Kira-Jennings**
Feature Flags 'Get Started' landing page updated to match [new designs]
(https://www.figma.com/file/9JdpTDJTjlLEYZiWwTioqd/FF---main?node-id=2155%3A61878) 
_Cosmetic only, no functionality has changed_

image

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 22 Issues 
* Code issue 22 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10913)

**Review by crystal-fong — DISMISSED** [review]
Looks good! excited to see this new shiny version 😄

File: src/modules/75-cf/pages/onboarding/OnboardingPage.module.scss [hunk 1]
```diff
@@ -5,84 +5,8 @@
...
-  }
-}
-
-.buttonConatiner {
-  display: grid !important;
-  justify-content: center;
-}
-
-.getStartedButton {
-  font-weight: 700 !important;
```
Comment (laura-mannus) [inline src/modules/75-cf/pages/onboarding/OnboardingPage.module.scss:87]: seeing all this go makes me so happy! Good job :)

File: src/modules/75-cf/pages/onboarding/OnboardingPage.tsx [hunk 2]
```diff
@@ -6,65 +6,68 @@
...
+                {getString('cf.onboarding.subtitle')}
+              </Text>
+              {OnboardingStepsDescription()}
+            </Layout.Vertical>
+            <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'flex-end' }}>
+              <Button
+                intent={Intent.PRIMARY}
+                variation={ButtonVariation.PRIMARY}
+                text={getString('cf.onboarding.tryItOut')}
+                size={ButtonSize.LARGE}
```
Comment (laura-mannus) [inline src/modules/75-cf/pages/onboarding/OnboardingPage.tsx:56]: oh wow this is cool, didn't know we had standardised button sizes!

**Review by laura-mannus — APPROVED** [review]
Amazing, great work

**Review by crystal-fong — APPROVED** [review]

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**